### PR TITLE
[FSSDK-10198] making isOdpIntegrated public

### PR DESCRIPTION
**Pull request by junaed-optimizely**
## Summary
Client should contain public facing API to provide information about ODP integration

## Test plan
Existing test should pass
## Issues
- [FSSDK-10198](https://jira.sso.episerver.net/browse/FSSDK-10198)

### CODE REVIEW

File: lib/optimizely/index.ts [hunk 1]
```diff
@@ -1723,7 +1719,7 @@ export default class Optimizely implements Client {
     }
   }
 
-  private isOdpIntegrated(): boolean {
+  public isOdpIntegrated(): boolean {
```
Comment (mikechu-optimizely) [inline lib/optimizely/index.ts:1722]: Now that we're making it public facing, can you add jsdoc?

**Review by mikechu-optimizely — CHANGES_REQUESTED** [review]
Small necessary change, but looks good.

File: lib/core/odp/odp_manager.ts [hunk 2]
```diff
@@ -241,7 +240,7 @@ export abstract class OdpManager implements IOdpManager {
     }
 
     if (!this.odpIntegrationConfig.integrated) {
-      this.logger.log(LogLevel.ERROR, ERROR_MESSAGES.ODP_NOT_INTEGRATED);
+      this.logger.log(LogLevel.INFO, ERROR_MESSAGES.ODP_NOT_INTEGRATED);
```
Comment (jaeopt) [inline lib/core/odp/odp_manager.ts:243]: This change looks good. We also need to change ERROR -> to INFO for "registerVuid()" api call as well.

**Review by jaeopt — CHANGES_REQUESTED** [review]
One required change

File: lib/optimizely/index.ts [hunk 3]
```diff
@@ -1718,7 +1718,10 @@ export default class Optimizely implements Client {
       this.logger.error(ERROR_MESSAGES.ODP_EVENT_FAILED, e);
     }
   }
-
+  /**
+   * Checks if ODP (Optimizely Data Platform) is integrated into the project.
+   * @returns { boolean }
```
Comment (mikechu-optimizely) [inline lib/optimizely/index.ts:1723]:
```suggestion
   * @returns { boolean } `true` if ODP settings were found in the datafile otherwise `false`
```

**Review by mikechu-optimizely — APPROVED** [review]
LGTM.

**Review by jaeopt — APPROVED** [review]
LGTM

**Pull request merged**